db/fixtures/paths/full_stack_rails/courses/ruby.rb

```diff
@@ -119,7 +119,7 @@
     ruby_lessons.fetch('Recursive Methods'),
     ruby_lessons.fetch('Recursion'),
     ruby_lessons.fetch('Common Data Structures and Algorithms'),
-    ruby_lessons.fetch('A Deeper Look at Git'),
+    git_lessons.fetch('A Deeper Look at Git'),
```

Comment:
What are your thoughts on having the two git lessons in their own dedicated section? Just to cut down on context switching a little. It could maybe be placed before the CS section?

Reply:
I totally get the feeling that keeping them in their own section might be a better approach. My initial thought was that having the git content spread throughout the course would make it more digestible and allow access to the first of the two advanced git lessons a bit earlier (since I don't think we want people doing Using Git in the Real World until they're near the end of each course).

That being said though keeping the git section may be the better call here as I think just about everyone on the team has had that reaction 😆